config/sprayproxy.yaml

```diff
@@ -38,6 +38,8 @@ spec:
           args:
             - "--secure-listen-address=0.0.0.0:8443"
             - "--upstream=http://127.0.0.1:6000/"
+            - "--upstream=http://127.0.0.1:8080/"
```

Comment:
@adambkaplan here is the problem. /metrics and other endpoints are listening on two different ports. It seems like we can't define two upstreams in `kube-rbac-proxy` container. Shouldn't we have same endpoints for metrics as well. WDYT? cc_ @xinredhat @bamachrn @rhopp

Reply:
Does adding two containers for `kube-rbac-proxy` makes sense?